packages/snapshot/package.json

```diff
@@ -43,13 +43,13 @@
     "prepublishOnly": "pnpm build"
   },
   "dependencies": {
+    "@vitest/utils": "workspace:*",
```

Comment:
in @vitest/snapshot utils should be in dev dependencies, it is bundled with the package

Reply:
No, it should be bundled

Reply:
`error` should be a different entry point `@vitest/utils/error` because it depends on diff which is a separate entry point

Reply:
in this PR there should be zero changes to packages/snapshot

move runner error to `@vitest/utils/error` and move `createSimpleStackTrace` into `base.ts`